src/routes/extras/containers/ExtrasContainer.js

```diff
@@ -52,11 +52,14 @@ class ExtrasContainer extends React.Component<PropsType, ExtrasStateType> {
     this.loadExtras()
   }
 
-  navigateToExtra = (path: string, isExternalUrl: boolean, offerHash: ?string = null) => {
+  navigateToExtra = (path: string, isExternalUrl: boolean, postData: ?Map<string, string>) => {
+    if (!this.props.navigation.push) {
+      return console.error('push is not defined on navigation')
```

Comment:
Is there a possibility that `push` is not defined? Or is this just a flow problem?

Reply:
That is a very weird problem with flow types of react-navigation. Couldn't fix it back then.

I would prefer:
```
throw Error('push is not defined on navigation')
```
which crashes the app and reports it though sentry.

Reply:
Ofc a real fix would be cool :)